src/VaultSharp/Properties/AssemblyInfo.cs

```diff
@@ -34,5 +34,4 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("0.6.1.0")]
 [assembly: AssemblyFileVersion("0.6.1.0")]
-[assembly: InternalsVisibleTo("VaultSharp.UnitTests")]
-[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
did the moq based tests run without this attribute? this attribute is needed for the moqing of internal classes.